### DISCUSSION TITLE
Configuring DNS

### DISCUSSION THREAD

**Discussion by stephanie-cristea** [community/community#23698] [Other Feature Feedback, Questions, & Ideas]
<p>Originally, I wanted to host my site on Google Sites and set up the DNS to do so, it worked. I’ve changed my mind and decided to reconfigure the DNS to use GitHub instead. I’ve followed the steps on <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site</a>, but my site <a href="http://stephaniecristea.me" rel="noopener nofollow ugc">stephaniecristea.me</a> is still throwing an error :confused:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/9/79c860cdc3ba831ec0db38dd976fa04391c22363.png" data-download-href="/uploads/short-url/hnl55COenbEZevvHPFmcGG0MERJ.png?dl=1" title="dns" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/22796419/181078802-b8a87a1e-7379-4e1b-ad39-908b1fa9e946.png" alt="dns" data-base62-sha1="hnl55COenbEZevvHPFmcGG0MERJ" width="690" height="116" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/9/79c860cdc3ba831ec0db38dd976fa04391c22363_2_690x116.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/9/79c860cdc3ba831ec0db38dd976fa04391c22363_2_1035x174.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/9/79c860cdc3ba831ec0db38dd976fa04391c22363_2_1380x232.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/9/79c860cdc3ba831ec0db38dd976fa04391c22363_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">dns</span><span class="informations">1490×252 11.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’d like to note that I’ve already gone in and added the site link to the repository on Github and removed the connection from the Google Sites.</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/1cristeaste">@1cristeaste</a>, welcome to the GitHub Support Community!</p>
<p>It looks like you’ve only set up your domain’s DNS records using the <code>www</code> subdomain, but you’ve instead attached the apex domain—the domain without the <code>www</code>—to your Pages site. To use the apex domain you’ll also need to set up the <code>A</code> records on the root domain—without the <code>www</code>.</p>
<p>To fix this you’ll need to add the <code>A</code> records listed in step 4 of the instructions here:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain" target="_blank" rel="noopener">Managing a custom domain for your GitHub Pages site - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by stephanie-cristea** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e26a83e8f570367d332ae8348fd0731ab4b5d88f.png" data-download-href="/uploads/short-url/wiY7cPNitXANG46Y3ga7BhvbvZl.png?dl=1" title="dns" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/22796419/181078807-158216a3-1368-4229-8fbd-8d71701fb955.png" alt="dns" data-base62-sha1="wiY7cPNitXANG46Y3ga7BhvbvZl" width="690" height="159" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e26a83e8f570367d332ae8348fd0731ab4b5d88f_2_690x159.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e26a83e8f570367d332ae8348fd0731ab4b5d88f_2_1035x238.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e26a83e8f570367d332ae8348fd0731ab4b5d88f_2_1380x318.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e26a83e8f570367d332ae8348fd0731ab4b5d88f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">dns</span><span class="informations">1388×321 16.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’ve followed the next step, waited a couple of hours (I heard it takes a couple of hours for the changes to trickle down) and it still doesn’t work. :confused:</p>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like you’ve set up the ALIAS record on that domain to point to itself rather than pointing it towards GitHub Pages. I’m not super familiar with ALIAS records but I believe you need to point them towards your GitHub address: <code>1cristeaste.github.io</code>.</p>
<p>That said I’d probably recommend removing it and adding the 4 A records that point towards GitHub IPs instead to guarantee it works.</p>
